code/game/objects/structures/tables_racks.dm

```diff
@@ -134,6 +134,8 @@
 	if(!ishuman(pushed_mob))
 		return
 	var/mob/living/carbon/human/H = pushed_mob
+	if(iscatperson(H))
```

Comment:
```suggestion
	if(iscatperson(H) && prob(50))
```

Reply:
considered it but id rather they just always nya when tabled because it's funny and not gameplay impacting